editor/sidebar/post-visibility/index.js

```diff
@@ -71,6 +74,21 @@ class PostVisibility extends Component {
 		this.setState( { opened: false } );
 	}
 
+	handleBlur( event ) {
+		if ( this.state.opened && event.relatedTarget !== null && ! this.dialog.contains( event.relatedTarget ) ) {
```

Comment:
Yes the setTimeout 0 trick is the only possible solution I've found so far, not so spectacularly happy to have to use setTimeout. I guess other solutions would be fragile though. Re: Firefox, I think at the time that comments were posted, Firefox was not supporting `relatedTarget`. It does now, since version 48 https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/relatedTarget
Instead, IE 11 could be a problem but I can't test in IE 11 at the moment 🙂  see #1397 and #1863